CONTRIBUTING.md

```diff
@@ -20,14 +21,19 @@ $ cd cppcache/test/<Debug|Release|if needed>
 $ ./apache-geode_unittests
 ```
 
-### Running old integration tests
+### Running integration tests
+There are two sets of integration tests based on different testing frameworks: CTest and [Google Test](https://github.com/google/googletest).
+
+CTest-based integration tests are deprecated. If your changes include the implementation of new integration test/s to be verified, they should be written using Google Test. If your change implies a significant change in one or more CTest-based test cases, you should create the equivalent test case/s using Google Test to substitute the old one/s instead of adapting the old ones.
```

Comment:
Again the deprecated old integration tests are written in a home grown framework. CTest is just a mechanism for managing and running tests in CMake. The new integration tests framework is home grown extension on top fo gtest. The unit tests are straight up gtest.